Makefile

```diff
@@ -8,7 +8,7 @@ GOLANGCILINT	?= $(shell command -v golangcli-lint 2>/dev/null)
 OUTPUT_DIRNAME	?= output/
 DOC_FILENAME	?= oci-distribution-spec
 
-PANDOC_CONTAINER ?= docker.io/vbatts/pandoc:1.19.1-3.fc27.x86_64
+PANDOC_CONTAINER ?= ghcr.io/jdolitsky/pandoc:aab258d@sha256:4772df87734a8c7cf1f4c8407a8b452ab3c9b250b425f27b657f9ca01d9a696b
```

Comment:
lets see if we can have the image in a joint repo. I had set up quay.io/oci as a group, but maybe ghcr.io makes more sense now. It should've been moved from my docker user a long time ago

Reply:
Are you fine if we copy this image into GHCR and/or Quay manually then figure out the automation parts later?

I dont have the Quay info, but I could probably get this uploaded into GHCR under opencontainers